test/settings.test.ts

```diff
@@ -8,4 +8,10 @@ describe("settings.json", () => {
       snapshot.utils.validateSchema(snapshot.schemas.space, settings),
     ).toBe(true);
   });
+  test("Spam protection: require 10000 COW/vCOW", () => {
+    expect(settings.validation.params.minScore == 10000).toBe(true);
+  });
+  test("Participation: require 35M COW/vCOW quorum", () => {
+    expect(settings.voting.quorum == 35000000).toBe(true);
+  });
```

Comment:
Nit for readability of tests and test output:

```suggestion
  test("Spam protection: require 10000 COW/vCOW", () => {
    expect(settings.validation.params.minScore).toBe(10_000);
  });
  test("Participation: require 35M COW/vCOW quorum", () => {
    expect(settings.voting.quorum).toBe(35_000_000);
  });
```

Reply:
Nice point!